### PR TITLE
Let know contributors they can add their login in contributors.txt

### PR DESCRIPTION
**Pull request by jrobinAV**
https://github.com/Avaiga/taipy-doc/issues/386

If it s fine and approved, I will propagate the change in all our repos.

### CODE REVIEW

**Comment by jrobinAV** [comment]
@FabienLelaquais Gentle reminder.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -102,8 +102,10 @@ working on it.
      pipenv run python -m pre-commit install
    ```
 
-4. Make the change and create a
-   [pull request from your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
+4. Make the change. You can also add your GitHub login in a new line of the `contributors.txt` file. We are using
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: =>
4. Make the changes.&lt;br/&gt;
&nbsp;&nbsp;&nbsp;You may want to also add your GitHub login as a new line of the \`contributors.txt\` file located at the root
&nbsp;&nbsp;&nbsp;of this repository. We are using it to list our contributors in the Taipy documentation (see the "Contributing > Contributors" section) and thank them.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -102,8 +102,10 @@ working on it.
...
+it to list our contributors on our website and thank them.
+
+5. Create a [pull request from your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: Add "&lt;br/&gt;" to actually break the line.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -102,8 +102,10 @@ working on it.
...
    Keep your pull request in __draft__ until your work is finished.
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: request in \_\_draft\_\_ => request as \_\_draft\_\_

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -102,8 +102,10 @@ working on it.
...
    Do not hesitate to add a comment for help or questions.
    Before you submit a pull request read to review from your forked repo, check that it meets these guidelines:
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: "Before you submit a pull request read to review"
Not sure this is English. Is it?

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
What do you think?

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**